api-module-library/microsoft-teams/manager.js

```diff
@@ -39,13 +46,14 @@ class Manager extends ModuleManager {
 
         if (params.entityId) {
             instance.entity = await Entity.findById(params.entityId);
+            instance.tenant_id = instance.entity.externalId;
```

Comment:
This is the main change in this file. The rest is just automated formatting fixes.